lib/matplotlib/tests/test_axes.py

```diff
@@ -1016,6 +1016,29 @@ def test_hexbin_log():
                   marginals=True, reduce_C_function=np.sum)
     plt.colorbar(h)
 
+    # Make sure offsets are set
+    assert h.get_offsets().shape == (11558, 2)
+
+
+def test_hexbin_log_offsets():
+    np.random.seed(19680801)
```

Comment:
Is there some randomness used here that I missed that we need to set the seed?

Reply:
No, I think just a hangover from a copy-pasted test. Fixed now 👍